EIPS/eip-5805.md

```diff
@@ -94,7 +118,11 @@ This function MUST be implemented
 
 #### getPastVotes
 
-This function returns the historical voting weight of an account. This corresponds to all the voting power delegated to it at a specific timepoint. The timepoint parameter should match the operating mode of the contract. This function SHOULD only serve past checkpoints that are immutable. Calling this function with a timepoint that is greater or equal to `clock()` SHOULD revert. For any timepoint that is strictly smaller than `clock()`, the value returned by `getPastVotes` should be constant.
+This function returns the historical voting weight of an account. This corresponds to all the voting power delegated to it at a specific timepoint. The timepoint parameter MUST match the operating mode of the contract. This function SHOULD only serve past checkpoints, which SHOULD be immutable.
+
+- Calling this function with a timepoint that is greater or equal to `clock()` SHOULD revert.
+- Calling this function with a timepoint scrictly smaller than `clock()` SHOULD not revert.
+- For any timepoint that is strictly smaller than `clock()`, the value returned by `getPastVotes` SHOULD be constant. This means that for any call to this function that returns a value, re-executing the same call (at any time in the future) SHOULD return the same value.
```

Comment:
"Timepoint" is not precisely defined, in particular nothing says all integers are valid timepoints, so I think it's better to specify "integer" here.

```suggestion
- For any integer that is strictly smaller than `clock()`, the value returned by `getPastVotes` SHOULD be constant. This means that for any call to this function that returns a value, re-executing the same call (at any time in the future) SHOULD return the same value.
```